sdk/metric/sdk.go

```diff
@@ -313,43 +339,34 @@ func New(batcher export.Batcher, opts ...Option) *SDK {
 		opt.Apply(c)
 	}
 
-	m := &SDK{
-		empty: labels{
-			ordered: [0]core.KeyValue{},
-		},
+	return &SDK{
 		batcher:      batcher,
 		errorHandler: c.ErrorHandler,
 		resource:     c.Resource,
 	}
-	m.empty.meter = m
-	m.empty.cachedValue = reflect.ValueOf(m.empty.ordered)
-	return m
 }
 
 func DefaultErrorHandler(err error) {
 	fmt.Fprintln(os.Stderr, "Metrics SDK error:", err)
 }
 
-// Labels returns a LabelSet corresponding to the arguments.  Passed labels
+// makeLabels returns a `labels` corresponding to the arguments.  Labels
 // are sorted and de-duplicated, with last-value-wins semantics.  Note that
 // sorting and deduplicating happens in-place to avoid allocation, so the
 // passed slice will be modified.
-func (m *SDK) Labels(kvs ...core.KeyValue) api.LabelSet {
+func (m *SDK) makeLabels(kvs []core.KeyValue, sortSlice *sortedLabels) labels {
```

Comment:
Can we add to the function documentation the significance of the passed `sortSlice` and how it is a placeholder for an allocation?

Reply:
Done.